apps/web/public/js/surveys.d.ts

```
@@ -0,0 +1,8 @@
+import { SurveyInlineProps } from '@formbricks/types/formbricks-surveys';
+import { SurveyModalProps } from '@formbricks/types/formbricks-surveys';
+
+export declare const renderSurveyInline: (props: SurveyInlineProps) => void;
+
+export declare const renderSurveyModal: (props: SurveyModalProps) => void;
+
+export { }
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)